specification/eventgrid/data-plane/Microsoft.AVS/preview/2023-09-01/examples/event-grid-schema/cluster_updated.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": "9b87886d-21a5-4af5-8e3e-10c4b8dac73b",
+  "eventType": "Microsoft.AVS.ClusterUpdated",
+  "topic": "/subscriptions/{subscription_id}/resourceGroups/{resource_group_name}/providers/Microsoft.AVS/privateClouds/{sddc_name}",
+  "data": {
+    "added": [
+      "esx17-r04.p01.431cc…"
+    ],
+    "removed": [
+      "esx15-r01.p01.431cc…"
+    ],
+    "maintenance": [
+      "esx01-r03.p01.431cc…"
+    ],
+    "operationId": "c8c0b304-2db3-402b-8a6b-06065e0e1755"
+  },
+  "subject": "/clusters/{cluster_name}",
```

Comment:
This may have been in the sample code, but is the caller expected to parse subject to get cluster_name, subscription_id, resource_group_name, and sddc_name?  Do you consider this to be regular enough to make it straightforward, or should they be well-defined nouns in the system.

Or, is this the existing style of other Event Grids, and we're following an established pattern.